src/components/Header/Header.scss

```diff
@@ -5,83 +5,70 @@
     width: 100%;
     height: 300px;
     position: relative;
+}
 
-    .header_cover {
-        width: 100%;
-        height: 100%;
-        background-image: url(moviecover.jpeg);
-        filter: blur(2px);
-        z-index: -1;
-
-        @include absoluteToTopLeft
-    }
-
-    .header_cover--black {
-        width: 100%;
-        height: 100%;
-        background-color: rgba(0, 0, 0, 0.4);
-
-        @include absoluteToTopLeft
-    }
-
-    .header_logo {
-        // position: absolute;
-        // top: 20px;
-        // left: 60px;
-
-        @include absoluteByArgument(20px, unset, unset, 60px);
-
-        color: $magentaPink;
-
-        span {font-weight: bold;}
-    }
-
-    .header_addMovieButton {
-        // position: absolute;
-        // top: 20px;
-        // right: 60px;
-
-        @include absoluteByArgument(20px, 60px);
+.header_cover {
+  @include absolute-to-top-left;
 
-        color: $magentaPink;
-        
-    }
+  width: 100%;
+  height: 100%;
+  background-image: url(moviecover.jpeg);
+  filter: blur(2px);
+  z-index: -1;
+}
 
-    .header_searchWrapper {
-        width: 80%;
-        height: 100%;
-        margin: auto;
-        display: flex;
-        flex-direction: column;
-        justify-content: center;
-        align-items: flex-start;
+.header_cover--black {
+  @include absolute-to-top-left;
 
-        h2 {
-                margin-top: 0;
-                text-transform: uppercase;
-                color: white;
-                font-weight: normal;
-            }
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.4);
+}
 
-        .header_searchWrapper_inputContainer {
-            width: 100%;
-            display: flex;
-            justify-content: center;
-            align-items: center;
+.header_logo {
+  @include absolute-by-argument(20px, unset, unset, 60px);
 
-            input {
-                width: calc(80% - 10px);
-                margin-right: 10px;
-            }
+  color: $magentaPink;
 
-            button {
-                width: 20%;
-                height: 35px;
-            }
-        }
-    }
+  span {font-weight: bold;}
 }
 
+.header_addMovieButton {
+  @include absolute-by-argument(20px, 60px);
 
+  color: $magentaPink;
+}
 
+.header_searchWrapper {
+  width: 80%;
+  height: 100%;
+  margin: auto;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+
+  h2 {
+    margin-top: 0;
+    text-transform: uppercase;
+    color: white;
+    font-weight: normal;
+  }
+}
 
+.header_searchWrapper_inputContainer {
```

Comment:
```suggestion
.header__search-wrapper__input-container {
```

Reply:
In general for BEM, consider the "dunder" :D `__` and `--` are doubled by purpose.